clay/gp.py

```diff
@@ -199,6 +199,14 @@ def from_data(cls, data, source, many=False):
 
         raise AssertionError()
 
+    def inc_playcount(self):
```

Comment:
I defer to @ValentijnvdBeek and @and3rson, but I would prefer ```increment_playcount```. It fits better stylistically.